.github/actions/spelling/allow.txt

```diff
@@ -0,0 +1,382 @@
+abhaykatheria
+ableabhinav
+accountsservice
+adriennefriend
+Affero
+Aissue
+Akechi
+amd
+ananthan
+anthonyharrison
+Aopen
+api
+apk
+Args
+asn
+autoescape
+autoextract
+backend
+backport
+bcca
+bdbd
+berenjena
+Bienvenido
+bigbird
+binutils
+bksahu
+blog
+bluetooth
+bluetoothctl
+bluez
+brainwane
+brotlipy
+btn
+bubblewrap
+bugfixes
+busybox
+bzip
+cabextract
+CDNs
+cfea
+cff
+chaitanyamogal
+charset
+checkername
+chris
+cleartext
+clnt
+cmdline
+codecov
+combintation
+config
+copyleft
+cpio
+cpp
+CQA
+crond
+cryptsetup
+csrc
+css
+csv
+customisation
+customise
+customising
+cve
+cvedb
+cvedetails
+cvss
+cyberciti
+cygwin
+dbus
+debian
+debuginfo
+delimted
+devops
+dgst
+distro
+dnsmasq
+docstring
+DOCTYPE
+documentaiton
+dpkg
+dsa
+dtls
+enscript
+entrypoint
+Eqt
+Everyone
+everytime
+evp
+exe
+executables
+expat
+extrated
+fde
+fedoraproject
+ffmpeg
+figcaption
+filepaths
+filterdiv
+firefox
+freeradius
+FReeshabh
+frontend
+fsck
+fsf
+ftp
+fuction
+gcc
+getenv
+gettext
+Giridhar
+github
+glibc
+gnomeshell
+gnupg
+gnutls
+Google
+gpgme
+grep
+GSo
+gsoc
+gstreamer
+gupnp
+Habilidad
+Hacktoberfest
+haproxy
+Harmandeep
+haxx
+hdf
+hideme
+HMAC
+hostapd
+href
+Hrmn
+html
+http
+httpd
+hunspell
+hur
+icecast
+icu
+ikeydoherty
+img
+imgur
+imsahil
+informaci
```

Comment:
Could you double-check where this occurs?  It looks like a potential typo rather than actual word.

Reply:
Ok sure.

So, `informaci` can be found in **`i18n.md`** on line 174 ( as a part of the word `información`, an Italian adaption(?) Wow, I didn't know that ). I think its essential to put this in **`allow.txt`** since spell-checker only catches English typos.